Modules/Login/Http/Controllers/LoginController.php

```diff
@@ -33,11 +34,9 @@ public function submitLogin(Request $request): RedirectResponse
     {
         $credentials = $request->only('email', 'password');
         $loggedIn = $this->loginService->attemptLogin($credentials);
-
-        if ($loggedIn['res']==1) {
+        if (!empty($loggedIn)) {
```

Comment:
Here you can check either $loggedIn is true or false.

Reply:
fixed